python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -639,6 +643,16 @@ def quantize(model: nn.Module,
         :param **export_kwargs: will be passed to torch.onnx.export function.
         :return:            A accelerated torch.nn.Module if quantization is sucessful.
         """
+        invalidInputError(precision in ['int8', 'fp16', 'bf16'],
+                          "Only support 'int8', 'bf16', 'fp16' now, "
+                          "no support for {}.".format(precision))
+        # device name might be: CPU, GPU, GPU.0, VPUX ...
```

Comment:
So the GPU device name will be platform specific? If a model is saved on one machine and later loaded on another machine, it may not work if the GPU name changes?

Reply:
Yeah, the GPU device name maybe platform specific. I will verify the save/load behavior of different platforms.

Reply:
I verified that, based on current code, the device value is always the default value ('CPU') when loading, that is, it always loads a CPU model , which does not meet our expectations.

I think we should save the device value when saving, and compile according to this value when loading. At the same time, we should provide a `device` parameter when loading, so that users can also load a model saved on GPU.1 into GPU.0 or CPU (there maybe no GPU.1 on the new machine).

Reply:
Have fixed this in PyTorch InferenceOptimizer, keras will be completed in next PR as I found  keras InferenceOptimizer has no `save`/`load` method up to now.